src/modules/mc_hover_thrust_estimator/MulticopterHoverThrustEstimator.cpp

```diff
@@ -191,26 +173,55 @@ void MulticopterHoverThrustEstimator::Run()
 			reset();
 		}
 
-		if (_was_valid) {
+		if (_valid) {
```

Comment:
I am not sure if `valid` is correct here, should it be `!valid`? If one just reads this here:
```
if(valid) {
   publishInvalidStatus();
   valid = false;
}
```
Personally, I would prefer the variable to be named something like `should_publish_as_invalid`. Otherwise, it does not make sense if you invert the variable after publishInvalidStatus().

Reply:
Upps, missed the outer else statement